Events/ShipTargetedEvent.cs

```diff
@@ -0,0 +1,82 @@
+﻿using EddiDataDefinitions;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace EddiEvents
+{
+    public class ShipTargetedEvent : Event
+    {
+        public const string NAME = "Ship targeted";
+        public const string DESCRIPTION = "Triggered when the player selects a target";
+        public const string SAMPLE = "{\"timestamp\":\"2018-05-09T23:19:49Z\",\"event\":\"ShipTargeted\",\"TargetLocked\":true,\"Ship\":\"adder\",\"ScanStage\":3,\"PilotName\":\"$npc_name_decorate:#name=Phoenix;\",\"PilotName_Localised\":\"Phoenix\",\"PilotRank\":\"Competent\",\"ShieldHealth\":100.000000,\"HullHealth\":100.000000,\"Faction\":\"Union Cosmos\",\"LegalStatus\":\"Lawless\",\"Subsystem\":\"$int_powerplant_size3_class3_name;\",\"Subsystem_Localised\":\"Power Plant\",\"SubsystemHealth\":100.000000}";
+        public static Dictionary<string, string> VARIABLES = new Dictionary<string, string>();
+
+        static ShipTargetedEvent()
+        {
+            VARIABLES.Add("targetlocked", "True if a ship has been targeted");
+            VARIABLES.Add("ship", "the model of the ship");
+            VARIABLES.Add("scanstage", "the stage of the ship scan");
+            VARIABLES.Add("name", "The name of the pilot");
+            VARIABLES.Add("rank", "The combat rank of the pilot");
```

Comment:
The journal describes this as the "pilot rank". Are you sure this isn't the highest rank the target has attained, rather than their combat rank?

Reply:
Did you look at the Journal Events to confirm? Used CombatRating object per your suggestion.

Reply:
Ahh ok. The journal gives a string (e.g. "Dangerous") rather than a number. Cool.